### DISCUSSION TITLE
Issue with the docs?

### DISCUSSION THREAD

**Discussion by henryiii** [PyO3/maturin#744] [Q&A] [answered]
In https://github.com/PyO3/maturin/blob/main/guide/src/tutorial.md#program-the-guessing-game-in-rust , the example doesn't seem to compile, I get:

```
error: argument never used=============> ] 42/43: cookie_maturin
  --> src/lib.rs:32:38
   |
32 |                 println!("You win!", guesses);
   |                          ----------  ^^^^^^^ argument never used
   |                          |
   |                          formatting specifier missing


error[E0425]: cannot find value `guesses` in this scopee_maturin
  --> src/lib.rs:32:38
   |
32 |                 println!("You win!", guesses);
   |                                      ^^^^^^^ help: a local variable with a similar name exists: `guess`


error: aborting due to 2 previous errors ] 42/43: cookie_maturin
```

This seems to need a format specifier & this variable isn't ever defined either. I'm guessing it was supposed to count the number of guesses? Also, "scopee" looks odd.

Also, if I compile this via PEP 517, it seems to miss this error message and reports something else.  https://github.com/scikit-hep/cookie/issues/60#issuecomment-992614803

**Comment by messense** [comment, marked as answer]
Thanks for reporting, I've fixed the doc in https://github.com/PyO3/maturin/commit/94f10af3c292b82f458cca56073e3a8c36292b91
